Resume playback automatically when wifi is available

rnauber opened the pull request:
If the playback was stopped on a cellular connection, it will resume automatically when the wifi is back. Before a manual click on reload was necessary.

rnauber commented:
Dear @cemrich,
I have tested this patch and it works for me. Is there anything else I can help to get this patch merged?

Best,
Richard

cemrich commented:
Hi @rnauber!
Thank you very much for your effort :) 

However, I removed this functionality intentionally because it can have severe side effects when playing in background. I received feedback from users that were woken up by resumed playbacks :)

As I currently have no resources to implement this "the proper way" (user friendly, taking edge cases into account), I'll not merge this - the current solution will work fine enough for most users.

rnauber commented:
> Hi @rnauber!
> Thank you very much for your effort :)
> 
> However, I removed this functionality intentionally because it can have severe side effects when playing in background. I received feedback from users that were woken up by resumed playbacks :)
> 
> As I currently have no resources to implement this "the proper way" (user friendly, taking edge cases into account), I'll not merge this - the current solution will work fine enough for most users.

Oh well, I can understand this. But the reason that I implemented this is that the slightest hiccup of my wifi will stop the playback and never resume.

What do you think of only resuming automatically, if the application is in foreground?


Best,
Richard
